tests/hipscat/catalog/association_catalog/test_association_catalog.py

```diff
@@ -4,47 +4,76 @@
 import pandas as pd
 import pytest
 
-from hipscat.catalog import CatalogType
+from hipscat.catalog import CatalogType, PartitionInfo
 from hipscat.catalog.association_catalog.association_catalog import AssociationCatalog
 from hipscat.catalog.association_catalog.partition_join_info import PartitionJoinInfo
+from hipscat.pixel_tree.pixel_node_type import PixelNodeType
 
 
-def test_init_catalog(association_catalog_info, association_catalog_join_pixels):
-    catalog = AssociationCatalog(association_catalog_info, association_catalog_join_pixels)
+def test_init_catalog(association_catalog_info, association_catalog_pixels, association_catalog_join_pixels):
+    catalog = AssociationCatalog(
+        association_catalog_info, association_catalog_pixels, association_catalog_join_pixels
```

Comment:
```suggestion
        association_catalog_info, [HealpixPixel(0,11)], association_catalog_join_pixels
```

I'm pretty sure you can just use a list of pixels for the input, instead of a dataframe. I think it makes tests more readable.